placebo/serializer.py

```diff
@@ -53,8 +66,13 @@ def serialize(obj):
         result['second'] = obj.second
         result['microsecond'] = obj.microsecond
         return result
-    if isinstance(obj, StreamingBody):
-        result['body'] = obj.read()
+    # Convert objects to dictionary representation based on type
+    if isinstance(obj, botocore.response.StreamingBody):
+        result['payload'] = obj.read() 
```

Comment:
In IoT ThingShadow responses, the payload is literally payload not body. We should probably support either/or?